.github/workflows/tests.yml

```
@@ -109,9 +109,7 @@ jobs:
       - name: Update pip
         run: python -m pip install -U pip
       - name: Install dependencies
-        # Temporarily hard-code the mypy version used for stubtest
-        # run: pip install $(grep tomli== requirements-tests.txt) (grep mypy== requirements-tests.txt)
-        run: pip install mypy==0.910
+        run: pip install $(grep tomli== requirements-tests.txt) (grep mypy== requirements-tests.txt)
```

For some reason I lost an `$`.

```suggestion
        run: pip install $(grep tomli== requirements-tests.txt) $(grep mypy== requirements-tests.txt)
```